nuclio/utils.py

```diff
@@ -117,6 +118,7 @@ def parse_export_line(args):
     parser.add_argument('--archive', '-a', action='store_true', default=False)
     parser.add_argument('--verbose', '-v', action='store_true', default=False)
     parser.add_argument('--kind', default=None)
+    parser.add_argument('--ignored-tags', default='')
```

Comment:
provide help for example?

Reply:
this is for the megic command, there is no help for support for it AFAIK